api/tests/unit/custom_auth/test_unit_custom_auth_views.py

```diff
@@ -141,3 +143,48 @@ def test_patch_user_onboarding_returns_error_if_tasks_and_tools_are_missing(
     assert response.json() == {
         "non_field_errors": ["At least one of 'tasks' or 'tools' must be provided."]
     }
+
+
+@pytest.mark.parametrize(
+    "hubspot_cookie",
+    [
+        "1234567890",
+        None,
+    ],
+)
+def test_create_user_calls_hubspot_tracking_and_creates_hubspot_contact(
+    mocker: MagicMock,
+    db: None,
+    settings: SettingsWrapper,
+    staff_client: APIClient,
+    hubspot_cookie: str,
+) -> None:
+    # Given
+    settings.ENABLE_HUBSPOT_LEAD_TRACKING = True
+    data = {
+        "first_name": "new",
+        "last_name": "user",
+        "email": "test@exemple.fr",
+        "password": "password123456!=&",
+        "sign_up_type": "NO_INVITE",
+        "hubspotutk": hubspot_cookie,
+    }
+
+    mock_create_hubspot_contact_for_user = mocker.patch(
+        "integrations.lead_tracking.hubspot.services.create_hubspot_contact_for_user"
+    )
+
+    url = reverse("api-v1:custom_auth:ffadminuser-list")
+    # When
+    response = staff_client.post(url, data=data, format="json")
+
+    user = FFAdminUser.objects.filter(email="test@exemple.fr").first()
+    hubspot_tracker = HubspotTracker.objects.filter(user=user).first()
+
+    # Then
+    assert response.status_code == status.HTTP_201_CREATED
+    assert user is not None
+    assert not hubspot_cookie or (
+        hubspot_tracker is not None and hubspot_tracker.hubspot_cookie == hubspot_cookie
+    )
```

Comment:
```suggestion
    assert not hubspot_cookie or hubspot_tracker.hubspot_cookie == hubspot_cookie
```

The `hubspot_tracker is not None` is redundant. If it happens to be `None`, the test will fail with an `AttributeError`.